src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.randomizer.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        int lengthOfArray = 6;
```

Comment:
I would recommend to make `lengthOfArray` a constant.

[Code recommendations](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)